src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -2073,6 +2073,51 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
         }
     }
 
+    public class NullValueToHorizontalAlignCenterConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
Using converter is overkill. Margin is UI element, and if it is possible, do it on UI. 

@Benglin you probably have more ideas.